aptos-move/framework/aptos-framework/sources/coin.spec.move

```diff
@@ -106,6 +106,21 @@ spec aptos_framework::coin {
             coin.value;
     }
 
+    spec schema AbortsIfAggregatorAdd<CoinType> {
```

Comment:
This looks good. Can we refactor this a little bit? To follow the convention:
(1) define the schema `AddAbortsIf` in the module `optional_aggregator`.
(2) include `AddAbortsIf` in the spec of `optional_aggregator::add` instead of using the existing `aborts_if` spec
(3) use (include) `AddAbortsIf` in the spec of `coin::mint` as well